consensus/src/simplex/actors/voter/state.rs

```diff
@@ -540,7 +541,7 @@ impl<E: Clock + Rng + CryptoRng + Metrics, S: Scheme<D>, L: Elector<S>, D: Diges
 mod tests {
     use super::*;
     use crate::simplex::{
-        elector::RoundRobin,
+        elector::RoundRobinConfig,
```

Comment:
nit: default `RoundRobinConfig` to being called `RoundRobin` and `RoundRobinElector` being constructed (because we use `RoundRobin` everywhere)?